deployments/charts/kuma/README.md

```diff
@@ -163,7 +163,7 @@ A Helm chart for the Kuma Control Plane
 | experimental.gatewayAPI | bool | `false` | If true, it installs experimental Gateway API support |
 | experimental.cni | bool | `false` | If true, it installs experimental new version of the CNI |
 | experimental.ebpf.enabled | bool | `false` | If true, ebpf will be used instead of using iptables to install/configure transparent proxy |
-| plugins.policies | list | `[]` |  |
```

Comment:
This seems like an odd way of expressing this value

Reply:
the _new_ way that is

Reply:
you mean this `plugins.policies[0] | string | "meshtrafficpermissions"`?

Reply:
Yeah

Reply:
Luckily there is a nice `@ignored` annotation for `helm-docs`